installer/bundle/bundle_skel.sh

```diff
@@ -805,6 +829,23 @@ if [ "$installMode" = "I" -o "$installMode" = "U" ]; then
         echo "sed was not installed, installation cannot continue. Please install sed."
         cleanup_and_exit $INSTALL_SED
     fi
+    curl_installed
+    if [ $? -ne 0 ]; then
+        if [ -z "${forceFlag}" ]; then
+            echo "Error: curl is not installed, installation cannot continue."
```

Comment:
The reason the python message has it is because `>&2` redirects to stderr. It's a small thing considering we print lots of warning/error messages to stdout instead. You've made all the lines consistent now, so it's not a big deal.